### PR TITLE
add kubeflow/fairing to prow

### PR DESCRIPTION
**Pull request by r2d4**
cc @jlewi @karthikv2k

### CODE REVIEW

**Comment by jlewi** [comment]
@r2d4 While you're here would you mind updating the OWNERs file to include some other folks from the team?
https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubeflow/OWNERS

**Comment by r2d4** [comment]
done @jlewi

File: config/jobs/kubeflow/OWNERS [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
+- abhi-g
 - jlewi
+- kunmingg
+- lluunn
+- richardsliu
+- texasmichelle
```
Comment (jlewi) [inline config/jobs/kubeflow/OWNERS]: nit; remove Michelle; I don't think she has context and I don't think she will be responsible for the test infra.

Comment (r2d4) [reply config/jobs/kubeflow/OWNERS]: done

**Pull request merged**